Genuine Protocol opt-in donation feature.rst

```diff
@@ -0,0 +1,130 @@
+::
+
+  ZIP: unassigned
+  Title: Genuine Protocol opt-in/out donation feature
+  Owner: unassigned
+  Advocate: mistfpga (zcash forums) <steve@mistfpga.net>
+  Category: Protocol
+  Created: 2019-07-17
+  License: CC BY-SA 4.0 (Creative Commons Attribution-ShareAlike 4.0) [1]
+
+Originally posted and discussed `on the Zcash Community Forum <https://forum.zcashcommunity.com/t/zip-proposal-a-genuine-opt-in-protocol-level-development-donation-option/33846>`__.
```

Comment:
This version appears to differ significantly from the one on the forum. It's unclear which is meant to be canonical, and the ZIP editors believe either would need additionaly specificity in terms of motivation and proposed protocol changes, if any. If no consensus protocol changes are required, as seems possible from our reading, then this should also be clearly stated.